contracts/ChildAccount.cdc

```diff
@@ -88,10 +94,15 @@ pub contract ChildAccount {
     /// its parent's ChildAccountManager
     ///
     pub resource ChildAccountTag : ChildAccountTagPublic {
```

Comment:
Still have a gripe about the naming here. How about ChildAccountHandler? Tag is non-verby, non-obvious

This also ties back to an earlier convo we had where we thought if we changed this to Handler then we should rename the other major types, review feedback for those also below

Reply:
I agree, and just made an issue (#6) specifically for the naming conventions. I didn't include it here because it will require updating all the named constructs in transactions & scripts and I just wanted to get some more comments into the contract in this repo & testnet for clarity.